<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VStrechTimer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color:rgb(0, 85, 150);
}
</string>
   </property>
   <widget class="QPushButton" name="startbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton
{
background-color:blue;
color:white;
border-radius:30px;
font-size:32px;
font-family:sans-serif;
}

QPushButton:hover
{
   background-color:white;
	color:black;
}

QPushButton:pressed
{
   background-color:yellow;
	color:black;
}</string>
    </property>
    <property name="text">
     <string>START TIMER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="killbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton
{
background-color:blue;
color:white;
border-radius:30px;
font-size:26px;
font-family:sans-serif;
font-weight:200
}

QPushButton:hover
{
   background-color:red;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>KILL PROCESS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:sans-serif;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
   </widget>
   <widget class="QLabel" name="launch_status">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:sans-serif;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string>Press &quot;Start timer&quot; to start.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:sans-serif;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string>Up Time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="launch_uptime">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:sans-serif;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string>COUNT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:sans-serif;
font-size:16px;
color:white;</string>
    </property>
    <property name="text">
     <string>Set Time Duration in mintues (Max: 50): </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="howmuch">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
border:4px solid white; 
border-radius:25px;
font-family:sans-serif;
font-size:25px;
color:white;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionStart"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Developer"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionAbout_Developer">
   <property name="text">
    <string>About Developer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
